### DISCUSSION TITLE
[Question][Hw5] Bug in test script

### DISCUSSION THREAD

**Discussion by jirheee** [prosyslab-classroom/cs524-program-analysis#334] [Q&A]
There is a strange bug with the `make test` command.

When I run `./analyzer [sign|interval] test/example5.ll`, it works well.

But when I run `make clean && make test` command, it fails. 

I tried to fix it with multiple ways, but I have no idea what is going on...

**Comment by minseongg** [comment]
Could you try again after `make clean && make` at `test` folder?

**Comment by jirheee** [comment]
Oh I'll try that. And I find one more strange behavior. The CI passes even though it doesn't pass in local environment..

**Comment by goodtaeeun** [comment]
Hi, we have checked your code and discovered the problem you have described.
You can also encounter the same problem when you run 
```
../analyzer [sign|interval] example5.ll
```
in the `test` directory.

However, despite this discrepancy, your code should be able to pass the test when run with `make test`.
We will also be looking into why this is happening, but please note that you still have to pass `make test`
